### DISCUSSION TITLE
Diatoms - Genomic DNA Extraction / Primers

### DISCUSSION THREAD

**Discussion by pdenuyl2** [NOAA-Omics/NOAA-Omics#14] [2-Lab]
We're starting to investigate diatoms in Lake Erie, Great Lakes.  Has anyone had experience extracting genomic DNA with a focus on diatoms?  We're planning on using the [Earth Microbiome Project: 18S Illumina Amplicon Protocol](https://earthmicrobiome.org/protocols-and-standards/18s/) primers, but are open to other suggestions.

**Comment by marinednadude** [comment]
Hey Paul - we have been using the following primers for phytoplankton in the Northeast Pacific:
|Primer Name|F primer|Reverse Primer|Citation|
|----|----|----|----------|
|Stoeck_phytoplankton_18S_V4|	CCAGCASCYGCGGTAATTCC	|ACTTTCGTTCTTGATYR	|"Stoeck, T., Bass, D., Nebel, M., Christen, R., Jones, M. D., BREINER, H. W., & Richards, T. A. (2010). Multiple marker parallel tag environmental DNA sequencing reveals a highly complex eukaryotic community in marine anoxic water. Molecular ecology, 19, 21-31."|
|AmaralZettler_phytoplankton_18S_V9	|TTGTACACACCGCCC|	CCTTCYGCAGGTTCACCTAC|	Amaral-Zettler, L. A., McCliment, E. A., Ducklow, H. W. & Huse, S. M. A method for studying protistan diversity using massively parallel sequencing of V9 hypervariable regions of small-subunit ribosomal RNA Genes. PLoS ONE 4, (2009).|
|WhiteSterling_phytoplankton_ITS1|	TCCGTAGGTGAACCTGCGG|	CATCCACCGCTGAAAGTTGTAA|	Sterling, A. R., Kirk, R. D., Bertin, M. J., Rynearson, T. A., Borkman, D. G., Caponi, M. C., ... & Jenkins, B. D. (2022). Emerging harmful algal blooms caused by distinct seasonal assemblages of a toxic diatom. Limnology and Oceanography, 67(11), 2341-2359.|

**Reply by pdenuyl2** [reply]
Thanks! This is really helpful. How do you decide which primers to use for a specific study? Do you evaluate the pros and cons outlined in the literature, or test different primers on the same samples in the lab?

**Reply by marinednadude** [reply]
@DiatomSetta Is leading our diatom efforts at PMEL

**Reply by marinednadude** [reply]
Choosing primers is a balance - so far along the West Coast we are exploring using all three marker sets to see which is the most valuable. Part of this choice is that both 18S marker sets are being used by the NOAA CalCOFI Ocean Genomics [NCOG](https://calcofi.org/data/marine-ecosystem-data/e-dna/) program and we wanted to eventually integrate our observations with there's. The ITS marker provides higher resolution for diatoms, particularly Psuedo-nitzschia species which are a major driver of HABs on the West Coast. In the Bering Sea and Arctic we decided to drop the 18Sv9 marker set since most work in the region used the 18Sv4 and we needed to balance costs.

Comes down to: 1) What is your question (e.g. which species do you most care about observing), 2) Do you want to integrate your data with existing observatories? , and 3) What can you afford to do? 

** 4) if there are comparisons that exist then this can inform 1), but often this information is not available
